media-api/app/lib/UsageQuota.scala

```diff
@@ -37,7 +40,7 @@ class UsageQuota(config: MediaApiConfig, elasticSearch: ElasticSearchVersion, sc
       waitMillis.millis)
   }.toOption.exists(_.exceeded) && FeatureToggle.get("usage-quota-ui")
 
-  def usageStatusForImage(id: String): Future[UsageStatus] = for {
+  def usageStatusForImage(id: String)(implicit request: AuthenticatedRequest[AnyContent, Principal]): Future[UsageStatus] = for {
```

Comment:
Any reason for this to be an implicit parameter rather than an explicit one?

Reply:
Minimising the change and preserving the explicit signature which was already there as this request awareness is a temporary thing.